docs/software/python-cli/installation.mdx

```diff
@@ -4,35 +4,40 @@ title: Meshtastic Python CLI installation
+## Meshtastic Python Library
 
-The [Meshtastic-python repo](https://github.com/meshtastic/Meshtastic-python) and [API documentation](https://python.meshtastic.org) are excellent sources of information.
+This library provides a command-line interface (CLI) for managing the user settings of Meshtastic nodes and provides an easy API for sending and receiving messages over mesh radios. Events are delivered using a publish-subscribe model, and you can subscribe to only the message types you are interested in.
 
-If you wish to view the code or contribute to development of the python library or the command line interface, please visit the Meshtastic python [GitHub page](https://github.com/meshtastic/Meshtastic-python).
+The [Meshtastic-python repo](https://github.com/meshtastic/Meshtastic-python) and [API documentation](https://python.meshtastic.org) are excellent sources of information. If you wish to view the code or contribute to the development of the Python library or the command-line interface, please visit the Meshtastic Python [GitHub page](https://github.com/meshtastic/Meshtastic-python).
 
-There are standalone executables for Windows and Ubuntu if you do not want to install python and/or the python libraries required to run the meshtastic CLI tool. See [Standalone](#standalone) for more information.
+### Installation Methods
 
-Installation can also be easily done through the [Python package installer pip](https://pypi.org/project/meshtastic):
-:::note
-You must use pip version 20 or later. To upgrade to the latest pip, do: `pip install --upgrade pip`
-:::
-:::info
-Make sure that the `PATH variable` also gets installed by checking the box while installing python. If you don't, python may not be available and you may not be able to call `meshtastic` from your CLI. If you do forget to check that box, you will need to install the path environment variable for python on your operating system.
-:::
-:::important
-You may need to install a driver from Silicon Labs for the [CP210X USB to UART bridge](https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers)
+You can install the Meshtastic CLI using one of the following methods:
+
+1. **Python Package Installer (pip)**: Installation can be easily done through the [Python package installer pip](https://pypi.org/project/meshtastic). Make sure to use pip version 20 or later by running `pip install --upgrade pip` if necessary.
+2. **Standalone Executable (Ubuntu only)**: A single executable file for Ubuntu is available on the [Releases](https://github.com/meshtastic/Meshtastic-python/releases) page. See [Standalone](#standalone-installation-ubuntu-only) below for instructions.
+
+### Prerequisites
```

Comment:
Do we typically put the prereqs at the end in our docs? Just wondering if it makes more sense to put them first so people don't proceed if they don't have these complete.

Reply:
Ah I see now! Yes a misunderstanding on my part.

I don't think either approach is wrong or right. If it's duplicate information, we just have to be sure it's useful to have it twice, IMO. You have more context/experience so I think you should make the call between those two.